distribution/ddf-common/src/main/resources/common-bin.xml

```diff
@@ -58,8 +58,8 @@
 
                 <!--
                 Skipping customized libraries.
-                -->
                 <exclude>**/org/apache/aries/blueprint/org.apache.aries.blueprint.core/1.8.2/org.apache.aries.blueprint.core-1.8.2.jar</exclude>
+                -->
```

Comment:
Do these comments need to be removed to turn on the Security Manager?  If so, we should probably explain that in the remarks above

Reply:
Honestly, I left them in with the expectation they'll be removed in the next week or two. Iff someone wanted to build a DDF with our security manager turned on they'd also have to include this hacked file; however, they'd also have to hack more files. This is here more as a hint on how to drop in the hacked files.

Usually, I'd rely on source control for things like this but in this one case I thought it made sense to keep the hint around for a little while.

Reply:
Should we create a ticket to get this removed and add a TODO here?